import main from './main'

main()
  .then(res => console.log(res))
  .catch(e => console.error(e))
